src/RA_Dlg_AchEditor.cpp

```diff
@@ -1897,7 +1897,7 @@ void Dlg_AchievementEditor::UpdateSelectedBadgeImage(const std::string& sBackupB
         sAchievementBadgeURI = sBackupBadgeToUse;
 
     m_hAchievementBadge.ChangeReference(ra::services::ImageType::Badge, sAchievementBadgeURI);
-    HBITMAP hBitmap = m_hAchievementBadge.GetHBitmap();
+    const HBITMAP__* hBitmap = m_hAchievementBadge.GetHBitmap();
```

Comment:
Why can't this be `const HBITMAP hBitmap`? I'd prefer to keep the exposed aliases, rather than rely on the underlying type that might get renamed at some point in the future.

Reply:
Well I can try suppressing it. Using `const HBITMAP` is the same as `HBITMAP__* const` which isn't what we need.